File: JAVA/Car.java
class Car{
    Integer id;
    String license;
    String driver;
    Integer passenger;

    public Car(String license, String driver){
        this.license = license;
        this.driver = driver ;
    }


    void printDataCar(){
        System.out.println("car licencia "+ license + " Driver "+ driver );
    }
}

File: PYTHON/car.py
class Car :
    def __init__(self):
        self.id          =   int
        self.license     =   str  
        self.driver      =   str
        self.passenger   =   int

    def printDataCar(self):
        print("Licensia " + self.license + " Driver " + self.driver)

